src/frontends/tensorflow_common/include/helper_ops/keep_in_graph_op.hpp

```diff
@@ -0,0 +1,38 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <vector>
+
+#include "helper_ops/internal_operation.hpp"
+
+namespace ov {
+namespace frontend {
+namespace tensorflow {
+
+class OPENVINO_API KeepInGraphOp : public InternalOperation {
+public:
+    OPENVINO_OP("KeepInGraphOp", "ov::frontend::tensorflow", InternalOperation);
+
+    KeepInGraphOp(const std::string& op_type_name, const std::shared_ptr<DecoderBase>& decoder = nullptr)
+        : InternalOperation(decoder, OutputVector{}, 1, op_type_name),
+          m_op_type_name(op_type_name) {}
```

Comment:
it should have one input. also, in translator you verify that it has one input. Let us use it and connect inputs with this node.

Reply:
Done.